esm/data.py

```diff
@@ -97,13 +97,15 @@ def __init__(
         prepend_bos: bool = True,
         append_eos: bool = False,
         use_msa: bool = False,
+        truncate: bool = True,
```

Comment:
better not modify the Alphabet class, `truncate` is only relevant for `BatchConverter`